ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableH2Database.scala

```diff
@@ -198,8 +200,8 @@ object EventsTableH2Database extends EventsTable {
       exerciseResults = serialized.exerciseResults,
     )
 
-    val archivals =
-      info.archives.iterator.map(archive(tx.offset)).toList
+    val archivals = Nil
```

Comment:
Where do we need this?

Reply:
This is just a cosmetic change to make it compile, all H2 support is broken at the moment.
There is a TODO capturing this aspect.